src/posts/2020-07-27-unsucking-your-project-onboarding.md

```diff
@@ -0,0 +1,789 @@
+---
+path: /blog/unsucking-your-project-onboarding
+title: "Unsucking your project onboarding: bin/ scripts"
```

Comment:
Is it just to me that the "bin/ script" part looks very confusing as part of the title?

Reply:
I agree with Naps. I usually refer to these as utility scripts.